src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogComputerListener.java

```diff
@@ -60,10 +60,18 @@ public class DatadogComputerListener extends ComputerListener {
     @Override
     public void onOnline(Computer computer, TaskListener listener) throws IOException, InterruptedException {
         try {
-            final boolean emitSystemEvents = DatadogUtilities.getDatadogGlobalDescriptor().isEmitSystemEvents();
-            if (!emitSystemEvents) {
+            final boolean canSendEvent = DatadogUtilities.shouldSendEvent("ComputerOnline");
```

Comment:
same remark here as in the function `shouldSendEvent`, can vs should.

Reply:
Updated to always be shouldSendEvent